src/compliment/sources/class_members.clj

```diff
@@ -152,40 +179,78 @@
        join
        (format "(%s)")))
 
+(defn- class&members->doc-dispatch [[^Class _cl members]]
+  (class (first members)))
+
+(defmulti ^:private class&members->doc #'class&members->doc-dispatch)
```

Comment:
The name looks kinda ugly with `&` and an arrow. I think you can drop the class and make it just `members->doc`. You can always extract the declaring class from the member.